### PR TITLE
Add integration tests for init command

### PR DESCRIPTION
**Pull request by NatashaWso2**

### CODE REVIEW

File: tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/packaging/PackagingInitTestCase.java [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+    @Test(description = "Test creating a project without going to interactive mode")
+    public void testInitWithoutGoingToInteractiveMode() throws Exception {
+        Path projectPath = tempProjectDirectory.resolve("testWithoutPackage");
+        Files.createDirectories(projectPath);
+
+        sourceFiles = new ArrayList<>();
+        packageDescFiles = new ArrayList<>();
+
+        sourceFiles.add(new SrcFile("", FileType.SERVICE));
+        packageDescFiles.add(new PackageMdFile("", FileType.SERVICE));
```
Comment (hemikak) [inline tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/packaging/PackagingInitTestCase.java]: Shall we create a constructor with not package name arg ? (the first arg)

File: tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/packaging/PackagingInitTestCase.java [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+
+        sourceFiles = new ArrayList<>();
+        packageDescFiles = new ArrayList<>();
+
+        sourceFiles.add(new SrcFile("newpkg", FileType.MAIN));
+        packageDescFiles.add(new PackageMdFile("newpkg", FileType.MAIN));
+
+        InitHandler.initialize(projectPath, manifest, sourceFiles, packageDescFiles);
+
+        Assert.assertEquals(Files.exists(projectPath.resolve("newpkg").resolve("main.bal")), true);
```
Comment (hemikak) [inline tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/packaging/PackagingInitTestCase.java]: We can use `Assert.assertTrue()` function here.

File: tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/packaging/PackagingInitTestCase.java (hunk 1 above)
Comment (NatashaWso2) [reply tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/packaging/PackagingInitTestCase.java]: Fixed with 6ff8577

File: tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/packaging/PackagingInitTestCase.java (hunk 2 above)
Comment (NatashaWso2) [reply tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/packaging/PackagingInitTestCase.java]: Fixed with 6ff8577

File: tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/packaging/PackagingInitTestCase.java [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+    private LogLeecher logLeecher;
+    private Path tempProjectDirectory;
+    private Manifest manifest;
+    private List<SrcFile> sourceFiles;
+    private List<PackageMdFile> packageDescFiles;
+
+    @BeforeClass()
+    public void setUp() throws BallerinaTestException, IOException {
+        serverZipPath = System.getProperty(Constant.SYSTEM_PROP_SERVER_ZIP);
+        tempProjectDirectory = Files.createTempDirectory("bal-test-integration-packaging-project-");
```
Comment (Kishanthan) [inline tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/packaging/PackagingInitTestCase.java]: Directory name ends with a hyphen  "bal-test-integration-packaging-project-" ?

Comment (NatashaWso2) [reply tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/packaging/PackagingInitTestCase.java]: No this directory name will be combined with a random number as below: 
Eg: bal-test-integration-packaging-project-5747492692079110850

Comment (Kishanthan) [reply tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/packaging/PackagingInitTestCase.java]: Ah ok.

File: tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/packaging/PackagingInitTestCase.java [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+
+        Assert.assertTrue(Files.exists(projectPath.resolve("foo").resolve("hello_service.bal")));
+        Assert.assertTrue(Files.exists(projectPath.resolve("Ballerina.toml")));
+        Assert.assertTrue(Files.exists(projectPath.resolve("foo").resolve("tests").resolve("hello_service_test.bal")));
+
+        ballerinaServerForService = ServerInstance.initBallerinaServer();
+        ballerinaServerForService.startBallerinaServer(projectPath.resolve("foo").resolve("hello_service.bal")
+                                                                  .toString());
+        HttpResponse response = HttpClientRequest.doGet(ballerinaServerForService.getServiceURLHttp
+                ("hello/sayHello"));
```
Comment (Kishanthan) [inline tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/packaging/PackagingInitTestCase.java]: This is not the correct way to break new lines. We usually break it with period (.).

**Comment by NatashaWso2** [comment]
Closing this PR infavor of [#9807](https://github.com/ballerina-platform/ballerina-lang/pull/9807)

**Pull request closed without merging**